volontulo/urls.py

```diff
@@ -41,6 +41,11 @@
         name='offer_form'
     ),
     url(
+        r'^offers/form/(?P<organization_id>[0-9]+)/(?P<offer_id>[0-9]+)$',
```

Comment:
Ok, I have found why, but it looks, that it will not wokr if You want to create new offer (if You don't have offer-id).